src/framework/audio/internal/synthesizers/fluidsynth/fluidsynth.cpp

```diff
@@ -112,7 +112,8 @@ Ret FluidSynth::init()
     fluid_settings_setint(m_fluid->settings, "synth.audio-channels", FLUID_AUDIO_CHANNELS_PAIR); // 1 pair of audio channels
     fluid_settings_setint(m_fluid->settings, "synth.lock-memory", 0);
     fluid_settings_setint(m_fluid->settings, "synth.threadsafe-api", 0);
-    fluid_settings_setint(m_fluid->settings, "synth.midi-channels", 80);
+    fluid_settings_setint(m_fluid->settings, "synth.midi-channels", 16);
```

Comment:
16 is very little for us :)

Reply:
it's enough for a single track, assuming that we have synth instance per track

Reply:
oh, sure